LMS/User/Models/Address.php

```diff
@@ -24,4 +24,18 @@ public function user(): BelongsTo
     {
         return $this->belongsTo(User::class);
     }
+
+    public function getFullAddress(): string
+    {
+        $fullAddress = $this->street.'-'.$this->number.'-'.$this->neighborhood;
```

Comment:
Pô, seria interessante colocar virgulas para separar essas partes do endereço, e deixar o - pra uma outra ocasião.

```suggestion
        $fullAddress = $this->street . ', ' . $this->number . ', ' . $this->neighborhood;
```